### PR TITLE
Account

### PR DESCRIPTION
**Pull request by Kanayo-can**
add new account section
add balance calculator

### CODE REVIEW

File: frontend/src/index.html [hunk 1]
```diff
@@ -69,15 +76,15 @@ <h1>Transaction</h1>
...
-            <th>Id</th>
-            <th>Username</th>
-            <th>Balance</th>
-          </tr>
-        </table> -->
+          <table id="as-table" class="table-auto">
+            <tr id="as-table-head" class="bg-slate-300">
+              <th class="w-20 text-right px-4 py2">Id</th>
+              <th class="px-4 py2">Username</th>
+              <th class="text-right px-4 py2" class="">Balance</th>
```
Comment (shylabo) [inline frontend/src/index.html]: All the `py2` should be `py-2`

File: frontend/src/js/components/Account.js [hunk 2]
```diff
@@ -32,7 +51,51 @@ $(document).ready(async function () {
...
+}
+
+// Add tailwind classes
+function setStyles() {
+  $('#as-table-head th').addClass('px-4 py2')
+  $('#as-table tr td:first-child').addClass('w-24 text-right px-4 py2')
+  $('#as-table tr td:nth-child(2)').addClass('px-4 py2')
+  $('#as-table tr td:last-child').addClass('px-4 py2 text-right')
+  $('#as-table tr:nth-child(odd)').addClass('bg-slate-200')
+}
```
Comment (shylabo) [inline frontend/src/js/components/Account.js]: Same as above. All the `py2` should be `py-2`

File: frontend/src/js/components/Account.js [hunk 3]
```diff
@@ -32,7 +51,51 @@ $(document).ready(async function () {
     // Account Summary
     // ==========================
     // Code here
+
+    fetchedAccounts.forEach(element => {
+
+      // call calcBalance
+      let balance = calcBalance(element)
+      balance = balance.toLocaleString("en-US", {style:"currency", currency:"CAD"});
```
Comment (shylabo) [inline frontend/src/js/components/Account.js:59]: Nice one!!

**Review by shylabo — COMMENTED** [review]
I left some comments regarding styles, but other than that, it looks good to me!

**Comment by Kanayo-can** [comment]
fixed class names and changed the color.
Not sure about the color but we will discuss and modify the styles later!

File: frontend/src/index.html [hunk 4]
```diff
@@ -28,7 +28,7 @@
             <h1 class="text-xl font-extrabold">Create your account</h1>
             <form class="flex flex-wrap justify-start items-center gap-4" id="na-account-form">
               <label class="font-bold" for="newAccount">New Account:</label>
-              <input class="p-3 rounded-lg" type="text" id="na-new-account" name="newAccount" placeholder="Mario" />
+              <input class="p-3 rounded-lg" comtype="text" id="na-new-account" name="newAccount" placeholder="Mario" />
```
Comment (shylabo) [inline frontend/src/index.html]: What is comtype?

File: frontend/src/js/components/Account.js [hunk 5]
```diff
@@ -95,7 +95,7 @@ function calcBalance(account) {
 function setStyles() {
   $('#as-table-head th').addClass('px-4 py2')
   $('#as-table tr td:first-child').addClass('w-24 text-right px-4 py2')
-  $('#as-table tr td:nth-child(2)').addClass('px-4 py2')
-  $('#as-table tr td:last-child').addClass('px-4 py2 text-right')
+  $('#as-table tr td:nth-child(2)').addClass('px-4 py-2')
+  $('#as-table tr td:last-child').addClass('px-4 py-2 text-right')
```
Comment (shylabo) [inline frontend/src/js/components/Account.js:99]: Thanks! There is two more left above.

**Review by shylabo — COMMENTED** [review]
Please check my comments.

File: frontend/src/index.html (hunk 4 above)
Comment (Kanayo-can) [reply frontend/src/index.html]: I have no idea... I will fix it!

**Comment by Kanayo-can** [comment]
I fixed mistype and classes and pushed again.
Please check them.

**Review by shylabo — APPROVED** [review]
LTGM

**Pull request merged**